styles/base.css

```diff
@@ -272,12 +296,6 @@
 
       --gradient-indigo-dark: var(--gradient-indigo-100);
       --gradient-indigo-light: var(--gradient-indigo-200);
-
-      --gradient-multitone: linear-gradient(
```

Comment:
Did we still need this alternative multi-tone gradient additional to multitone-pink? `linear-gradient(271deg, #EB1000 -38.76%, #4B75FF 118.78%);`

Reply:
Yes, but not in dark mode